fastlane_bot/helpers/txhelpers.py

```diff
@@ -813,21 +818,22 @@ def _get_layer_one_gas_fee_loop(self, rawTransaction) -> Decimal:
             self.ConfigObj.GAS_ORACLE_CONTRACT.caller.l1FeeScalar()
         ))
 
-        return self._get_layer_one_gas_fee(rawTransaction, ethereum_base_fee, fixed_overhead, dynamic_overhead)
-
-    def _get_layer_one_gas_fee(self, rawTransaction, ethereum_base_fee: int, fixed_overhead: int, dynamic_overhead: int) -> Decimal:
-        """
-        Returns the expected layer one gas fee for a layer 2 Optimism transaction
-        :param rawTransaction: the raw transaction
-        :param ethereum_base_fee: the L1 base fee received from the contract
-        :param fixed_overhead: the fixed overhead received from the contract
-        :param dynamic_overhead: the dynamic fee received from the contract
-        returns: Decimal
-            The total fee (in gas token) for the l1 gas fee
-        """
```

Comment:
This change can be reverted, as it will be obtained directly from a similar task already completed for Base and Optimism (which would otherwise lead to manual conflict when this change is merged).